src/ci/scripts/install-tidy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This script downloads and installs the tidy binary from Homebrew.
+
+set -euo pipefail
+IFS=$'\n\t'
+
+source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
+
+# Only the macOS arm64/aarch64 GitHub Actions runner needs to have tidy
+# installed; other platforms have it preinstalled.
```

Comment:
Q: I wonder if we can install this on the image build directly.

Reply:
[Not yet](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra/topic/Github.20M1.20CI.20seems.20to.20exist.20now/near/394881350)